File: src/commands/debug.ts
import { commands } from "./bindings";

// Fire-and-forget trace into ~/monica/logs/monica.log (issue #157). The release webview has no
// devtools, so console.log is invisible there; this routes through Rust's tauri-plugin-log sink.
export function debugLog(message: string): void {
  void commands.debugLog(message);
}

File: crates/monica-app/src/commands/debug.rs
/// Temporary diagnostic sink so the release webview (which has no devtools) can route
/// frontend trace points into ~/monica/logs/monica.log. Tracked in issue #157; remove
/// once the cmd+r PR-sync investigation concludes.
#[tauri::command]
#[specta::specta]
pub fn debug_log(message: String) {
    log::info!(target: "monica_app::debug", "{message}");
}
